sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -385,34 +385,29 @@ def list_accelerators(
             new_results[acc_name] = acc_info
     results = new_results
 
-    # Unlike other GPUs that can be attached to different sizes of N1 VMs,
-    # A100 GPUs can only be attached to fixed-size A2 VMs,
-    # and L4 GPUs can only be attached to G2 VMs.
-    # Thus, we can show their exact cost including the host VM prices.
-
-    acc_infos: List[common.InstanceTypeInfo] = sum(
-        [results.get(a, []) for a in _ACC_INSTANCE_TYPE_DICTS], [])
-    if not acc_infos:
-        return results
+    # If we can figure out which instances are to be used then we
+    # can attach them here.
+    acc_infos: List[common.InstanceTypeInfo] = sum(results.values(), [])
 
     new_infos = defaultdict(list)
     for info in acc_infos:
         assert pd.isna(info.instance_type) and pd.isna(info.memory), acc_infos
-        vm_types = _ACC_INSTANCE_TYPE_DICTS[info.accelerator_name][
-            info.accelerator_count]
+        vm_types, _ = get_instance_type_for_accelerator(info.accelerator_name,
```

Comment:
So we should pass `use_spot=True` ?

Reply:
We should pass `use_spot=True/False` twice and somehow merge the two lists. If one instance type is shown in both list, we show a single row in the final table with two prices; Otherwise we could left one of the column blank in the row.

Reply:
Seems like the case is not likely to happen; lets just remove the `use_spot=True` in the latter comments